esx_service/cli/vmdkops_admin.py

```diff
@@ -1006,15 +1006,46 @@ def tenant_access_add(args):
         error_info = "Tenant {0} does not exist".format(args.name)
         return operation_fail(error_info)
 
+    privileges = [d for d in tenant.privileges if d[auth_data_const.COL_DATASTORE] == args.datastore]
```

Comment:
And, more importantly, this seems to contradict https://github.com/vmware/docker-volume-vsphere/pull/729 and will generate conflict. I thoink the checks in VMODL and CLI should be the same, so it should be in the API layer. Please branch off #729